src/aspire/source/xform.py

```diff
@@ -82,6 +82,13 @@ def disabled(self):
         """
         return Xform.XformActiveContextManager(self, active=False)
 
+    def __str__(self):
+        """
+        Show class name of Xform
```

Comment:
Don't add docstrings to dunder methods unless there's something really non-intuitive going on inside them.

Reply:
Removed them in the base class. The derive classes also print out other information so the docstrings still there.

Reply:
Removed docstrings for all __str__ methods in new commit 2c901f7.